test/boilerplate_suggestors/factory_ignore_comment_mover.suggestor_test

```diff
@@ -0,0 +1,41 @@
+FactoryIgnoreCommentMover @dartfmt_output
```

Comment:
I don't know if it matters if these tests aren't in written in dart like the rest of the boilerplate tests?

Reply:
It's how suggestor tests used to be written, and I went with it because it was easier and I was feeling lazy haha.